pkg/testframework/framework.go

```diff
@@ -137,6 +139,17 @@ func DeleteCompletely(getObject func() (runtime.Object, error), deleteObject fun
 	})
 }
 
+func ConsoleResourcesAvailable(client *Clientset) error {
```

Comment:
Cool.  Technically we do have `2` `configmap`s, and then a whole additional `Deployment,Service,Route` for downloads, so we may at some point want to work that into our helpers, but not important now.